manifests/repo.pp

```diff
@@ -188,7 +188,7 @@
         logoutput => on_failure,
       }
 
-      if $::mrepo_env {
+      if $mrepo_env {
```

Comment:
@petems This 'fix' might actually cause problems.  I'm still investigating.  At the moment, the class parameter defaults to `''` (empty string).  In an old version of puppet, `'' == false` but now is `true`?  I'd like to look a bit closer before merging.